pkg/apis/events/v1alpha1/cloudauditlogssource_conversion_test.go

```diff
@@ -35,85 +33,14 @@ import (
 // These variables are used to create a 'complete' version of CloudAuditLogsSource where every field is
 // filled in.
 var (
-	trueVal = true
-	seconds = int64(314)
-
-	completeObjectMeta = metav1.ObjectMeta{
-		Name:            "name",
-		GenerateName:    "generateName",
-		Namespace:       "namespace",
-		SelfLink:        "selfLink",
-		UID:             "uid",
-		ResourceVersion: "resourceVersion",
-		Generation:      2012,
-		CreationTimestamp: metav1.Time{
-			Time: time.Unix(1, 1),
-		},
-		DeletionTimestamp: &metav1.Time{
-			Time: time.Unix(2, 3),
-		},
-		DeletionGracePeriodSeconds: &seconds,
-		Labels:                     map[string]string{"steel": "heart"},
-		Annotations:                map[string]string{"New": "Cago"},
-		OwnerReferences: []metav1.OwnerReference{
-			{
-				APIVersion:         "apiVersion",
-				Kind:               "kind",
-				Name:               "n",
-				UID:                "uid",
-				Controller:         &trueVal,
-				BlockOwnerDeletion: &trueVal,
-			},
-		},
-		Finalizers:  []string{"finalizer-1", "finalizer-2"},
-		ClusterName: "clusterName",
-	}
-
-	completeURL = apis.URL{
-		Scheme:     "scheme",
-		Opaque:     "opaque",
-		User:       url.User("user"),
-		Host:       "host",
-		Path:       "path",
-		RawPath:    "rawPath",
-		ForceQuery: false,
-		RawQuery:   "rawQuery",
-		Fragment:   "fragment",
-	}
-
-	completeDestination = duckv1.Destination{
-		Ref: &duckv1.KReference{
-			APIVersion: "apiVersion",
-			Kind:       "kind",
-			Namespace:  "namespace",
-			Name:       "name",
-		},
-		URI: &completeURL,
-	}
-
-	completeSourceSpec = duckv1.SourceSpec{
-		Sink: completeDestination,
-		CloudEventOverrides: &duckv1.CloudEventOverrides{
-			Extensions: map[string]string{"supers": "reckoners"},
-		},
-	}
-
 	completeIdentitySpec = duckv1alpha1.IdentitySpec{
 		ServiceAccountName: "k8sServiceAccount",
 	}
 
-	completeSecret = &v1.SecretKeySelector{
-		LocalObjectReference: v1.LocalObjectReference{
-			Name: "name",
-		},
-		Key:      "key",
-		Optional: &trueVal,
-	}
-
 	completePubSubSpec = duckv1alpha1.PubSubSpec{
```

Comment:
I found that you defined the completePubSubSpec in v1beta1/test_helper.sh, and all conversion_test in v1beta1 can use that, which is nice. However, in v1alpha1, why we still define the completePubsubSpec in the auditlogsSource conversion_test? Will that be inconsistent with what you did in v1beta1?

Reply:
I have not refactored that part, we can do that in follow up PRs since it's only related to v1alpha1 and it's just a clean up